packaging_automation/tests/test_publish_docker.py

```diff
@@ -16,10 +16,10 @@
     publish_nightly_docker_image,
 )
 
-NON_DEFAULT_BRANCH_NAME = "10.0.3_test"
+NON_DEFAULT_BRANCH_NAME = "12.0.0_test"
 DEFAULT_BRANCH_NAME = "master"
-TAG_NAME = "v10.0.3"
-INVALID_TAG_NAME = "v10.x"
+TAG_NAME = "v12.0.0"
```

Comment:
Why do you not want to use the latest version 12.1 here?

Reply:
not released yet :) To release the new docker images I need this development